src/main/java/com/softdevsix/api/controllers/CoverageNotificationController.java

```diff
@@ -0,0 +1,44 @@
+package com.softdevsix.api.controllers;
+
+import com.softdevsix.api.repositories.client.IFileManagerClientRepository;
+import com.softdevsix.api.services.report.IReportService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/coverageNotification")
```

Comment:
Please rename endpoint to coverage/creation-notification. To follow the documentation and reduce issues.

Reply:
Done. 
signed by Salet